### PR TITLE
Fixup ETB auto-calib/tune buttons

### PR DESCRIPTION
**Pull request by nmschulte**
for #5061

### CODE REVIEW

**Comment by nmschulte** [comment]
I don't immediately have access to hardware to test this.

File: firmware/controllers/actuators/electronic_throttle.cpp [hunk 1]
```diff
@@ -447,19 +447,15 @@ expected<percent_t> EtbController::getClosedLoopAutotune(percent_t target, perce
 		float kd = 0.08f * ku * m_tu;
 
 		// Every 5 cycles (of the throttle), cycle to the next value
-		if (m_autotuneCounter == 5) {
+		if (m_autotuneCounter >= 5) {
```
Comment (mck1117) [inline firmware/controllers/actuators/electronic_throttle.cpp:450]: the idea with the `>` was that it's more robust against an extra `++` slipped in somewhere, but it's not a big deal.

Comment (nmschulte) [reply firmware/controllers/actuators/electronic_throttle.cpp:450]: That's why I added it ;)

File: firmware/tunerstudio/rusefi.input [hunk 2]
```diff
@@ -4028,11 +4028,11 @@ dialog = tcuControls, "Transmission Settings"
...
-		commandButton = "Auto Calibrate ETB 2", cmb_etb_auto_calibrate_2, {etb2configured == 1 && tps2_1AdcChannel != @@ADC_CHANNEL_NONE@@}
+		commandButton = "Auto Calibrate ETB 1", cmd_etb_auto_calibrate, {etb1configured == 1 && tps1_1AdcChannel != @@ADC_CHANNEL_NONE@@ && calibrationMode == @@TsCalMode_None@@}
+		commandButton = "Auto Calibrate ETB 2", cmd_etb_auto_calibrate_2, {etb2configured == 1 && tps2_1AdcChannel != @@ADC_CHANNEL_NONE@@ && calibrationMode == @@TsCalMode_None@@}
 		field = "Second step"
-		commandButton = "Start ETB PID Autotune", cmd_etb_autotune,       { etb1configured == 1 || etb2configured == 1 }
-		commandButton = "Stop ETB PID Autotune",  cmd_etb_autotune_stop,  { etb1configured == 1 || etb2configured == 1 }
+		commandButton = "Start ETB PID Autotune", cmd_etb_autotune,       {(etb1configured == 1 || etb2configured == 1) && calibrationMode = @@TsCalMode_None@@}
+		commandButton = "Stop ETB PID Autotune",  cmd_etb_autotune_stop,  {(etb1configured == 1 || etb2configured == 1) && calibrationMode >= @@TsCalMode_EtbKp@@ && calibrationMode <= @@TsCalMode_EtbKd@@}
```
Comment (nmschulte) [inline firmware/tunerstudio/rusefi.input:4035]: The condition in parenthesis is redundant now, as 3-5 imply ETB calib underway.  It also may cause a problem if somehow the calibration is enabled and they change: cannot be disabled (though that's always been the case): shall I remove it?

File: firmware/controllers/actuators/electronic_throttle.cpp (hunk 1 above)
Comment (mck1117) [reply firmware/controllers/actuators/electronic_throttle.cpp:450]: oh you ADDED it, apparently I can't read a diff

**Pull request merged**